_posts/2017-04-01-docker-flow-proxy.markdown

```diff
@@ -0,0 +1,505 @@
+---
+layout: post
+title:  "WebApps with Docker Flow Proxy"
+date:   2017-04-01 10:51:47 +0530
+author: "Sébastien Allamand (allamand)"
+categories: webapps
```

Comment:
`categories` should be `category` and needs to be one of ["beginner", "intermediate", "advanced"]